src/Rest/RestApiBrowser.php

```diff
@@ -138,6 +148,7 @@ public function setRequestHeader($name, $value)
      */
     public function addRequestHeader($name, $value)
     {
+        $name = strtolower($name);
```

Comment:
I made header's name lower case, because I have to be sure if content-type header has 'multipart/form-data' value.